atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServices.java

```diff
@@ -59,6 +59,7 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
+@SuppressFBWarnings("SLF4J_ILLEGAL_PASSED_CLASS")
```

Comment:
There were a few places where I suppressed this warning because the comments in the atlas code indicated the existing logger class name is not a typo. Don't have strong feelings either way.